packages/controllers/file_upload.coffee

```diff
@@ -0,0 +1,19 @@
+Template.fileUpload.onCreated ->
+  @fileName = new ReactiveVar('No file selected')
+  @hasFile = new ReactiveVar(false)
+
+Template.fileUpload.helpers
+  fileName: ->
+    Template.instance().fileName.get()
+  hasFile: ->
+    if Template.instance().hasFile.get()
+      'has-file'
+
+Template.fileUpload.events
+  'change .btn-file': (event, template) ->
+    template.fileName.set(event.target.value.replace('C:\\fakepath\\',''))
```

Comment:
No, its added for security reasons and mandated by W3C.
http://dev.w3.org/html5/spec-preview/common-input-element-apis.html#dom-input-value-filename